cmake/config.cmake

```diff
@@ -146,8 +144,20 @@ set(OMR_GC_SCAN_OBJECT_GLUE OFF CACHE BOOL "Implement ScanObject in glue code, n
 set(OMR_GC_SEGREGATED_HEAP OFF CACHE BOOL "TODO: Document")
 set(OMR_GC_VLHGC OFF CACHE BOOL "TODO: Document")
 set(OMR_GC_VLHGC_CONCURRENT_COPY_FORWARD OFF CACHE BOOL "Enable VLHGC concurrent copy forward")
-
 set(OMR_INTERP_HAS_SEMAPHORES ON CACHE BOOL "TODO: Document")
+set(OMR_GC_POINTERS "full" CACHE BOOL "The mode of pointers.  Must be \"full\", \"compressed\", or \"mixed\"")
```

Comment:
This is not a bool.

Reply:
It should be a string. You can set the STRINGS property to document the possible values. https://cmake.org/cmake/help/latest/prop_cache/STRINGS.html#prop_cache:STRINGS